source/page/Map/component/EChartsMap.tsx

```diff
@@ -92,13 +94,20 @@ export class EChartsMap
                     hovered = '';
                 }
             })
-            .on('click', 'timeline', ({ dataIndex }) =>
+            .on('click', 'timeline', async ({ dataIndex }) => {
+                const formattedDate = formatDate(dataIndex, 'YYYY-MM-DD');
                 chart.dispatchAction({
                     type: 'timelineChange',
                     // index of time point
                     currentIndex: data.findIndex(d => d === dataIndex)
-                })
-            );
+                });
+                try {
+                    const newData = await getHistory(formattedDate);
+                    this.updateChartData(newData);
+                } catch (error) {
+                    console.error('Failed to fetch data:', error);
+                }
```

Comment:
```suggestion
                const newData = await getHistory(formattedDate);

                this.updateChartData(newData);
```
时间轴日期只要不超出范围，按理说应该所有数据都能正常请求到，不会有网络错误，有的话现在也会全局统一捕获并提示。